EnglishFilter/sections/css_extended.txt

```diff
@@ -8,6 +8,9 @@ lovehug.net#?#center > h5 > font[color="white"]:contains(Advertisement)
 ! https://github.com/AdguardTeam/AdguardFilters/issues/73691
 mad4wheels.com#?#.prodotti-grid > .mb-5 > .card > .card-body > a > img[src^="/ads/"]:upward(.mb-5)
 mad4wheels.com#?#.prodotti-grid > .mb-5 > .card  > .card-footer > .pull-right:contains(/^Ads$/):upward(.mb-5)
+! https://github.com/AdguardTeam/AdguardFilters/issues/74335
+linkedin.com#?##main > div[id^="ember"] > div > div[data-id][class="relative ember-view"]:has( li.artdeco-carousel__item > div.artdeco-carousel__item-container > div.ember-view > a[target="_blank"][rel="noopener nofollow"])
+linkedin.com#?##main > div[id^="ember"] > div > div[data-id][class="relative ember-view"]:has( li.artdeco-carousel__item > div.artdeco-carousel__item-container > div.ember-view > a[href^="https://www.linkedin.com/leadGenForm/"])
```

Comment:
The problem is that these rules do not hide sponsored posts on my side.
There is no `a[href^="https://www.linkedin.com/leadGenForm/"]` and `a[target="_blank"][rel="noopener nofollow"]` on my end.

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/108200558-eaf7d480-711e-11eb-93eb-ea7dd6ed8749.png)

</details>

This rule works fine:

```
linkedin.com#?##main > .ember-view > div > div[data-id^="urn:li:activity:"] > .occludable-update > div[class^="feed-shared-update-"][data-urn] > div > .feed-shared-actor > .app-aware-link > .feed-shared-actor__meta > span.feed-shared-actor__sub-description:last-child:not(:has(>*)):upward(div[data-id^="urn:li:activity:"])
```

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/108200645-082ca300-711f-11eb-9ed4-913f5e1d1f10.png)

</details>

But I'm not sure if it's safe to add.

Reply:
There's a chance that these promoted posts depend on you region and profile content

Reply:
Yes it may depends on that, so I think that we should add rule(s) which will work independently of the region and profile content.

Reply:
This works on my side atm, swapped

Reply:
Okay, if it will cause incorrect blocking then I think we will need to add `:contains(/Promoted|promoted text in another languages/)` to `span.feed-shared-actor__sub-description`